src/validation/validate.ts

```diff
@@ -95,6 +97,19 @@ export function validate(options: BeachballOptions, validateOptionsOverride?: Pa
     }
   }
 
+  if (!isChangeNeeded) {
+    const bumpInfo = gatherBumpInfo(options);
```

Comment:
I think `gatherBumpInfo` will bump package versions in place (write to package.json). can you double check?

Reply:
I tried it out and it didn't cause any local changes.

Reply:
Would be ideal to also add a test for this whole scenario (including that it doesn't modify any files) but I'm not sure how hard that would be...

Reply:
Ok, my bad - `performBump` is the one actually writes.